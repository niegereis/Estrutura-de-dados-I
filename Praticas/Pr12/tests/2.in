A fictional narrative
---------------------
Fictionality is most commonly cited as distinguishing novels from historiography. However this can be a problematic criterion. Throughout the early modern period authors of historical narratives would
often include inventions rooted in traditional beliefs in order to embellish a passage of text or add credibility to an opinion. Historians would also invent and compose speeches for didactic
purposes. Novels can, on the other hand, depict the social, political and personal realities of a place and period with clarity and detail not found in works of history.

Literary prose
--------------
While prose rather than verse became the standard of the modern novel, the ancestors of the modern European novel include verse epics in the Romance language of southern France, especially those by
Chretien de Troyes (late 12th century), and in Middle English (Geoffrey Chaucer's (c. 1343 â€“ 1400) The Canterbury Tales).[8] Even in the 19th century, fictional narratives in verse, such as Lord
Byron's Don Juan (1824), Alexander Pushkin's Yevgeniy Onegin (1833), and Elizabeth Barrett Browning's Aurora Leigh (1856), competed with prose novels. Vikram Seth's The Golden Gate (1986), composed of
590 Onegin stanzas, is a more recent example of the verse novel.

Content: intimate experience
----------------------------
Both in 12th-century Japan and 15th-century Europe, prose fiction created intimate reading situations. On the other hand, verse epics, including the Odyssey and Aeneid, had been recited to a select
audiences, though this was a more intimate experience than the performance of plays in theaters.

A new world of Individualistic fashion, personal views, intimate feelings, secret anxieties, "conduct" and "gallantry" spread with novels and the associated prose-romance.

Length
------
The novel is today the longest genre of narrative prose fiction, followed by the novella, short story, and flash fiction. However, in the 17th century critics saw the romance as of epic length and the
novel as its short rival. A precise definition of the differences in length between these types of fiction, is, however, not possible.The requirement of length has been traditionally connected with
the notion that a novel should encompass the "totality of life."[10]

The length of a novel can still be important because most literary awards use length as a criterion in the ranking system.
