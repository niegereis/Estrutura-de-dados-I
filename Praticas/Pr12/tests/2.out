9
a 12
add 1
aeneid 1
alexander 1
also 1
an 1
ancestors 1
and 12
anxieties 1
as 5
associated 1
audiences 1
aurora 1
authors 1
awards 1
barrett 1
be 2
became 1
because 1
been 2
beliefs 1
between 1
both 1
browning 1
by 2
byron 1
c 1
can 3
canterbury 1
century 5
chaucer 1
chretien 1
cited 1
clarity 1
commonly 1
competed 1
compose 1
composed 1
conduct 1
connected 1
content 1
created 1
credibility 1
criterion 2
critics 1
de 1
definition 1
depict 1
detail 1
didactic 1
differences 1
distinguishing 1
don 1
early 1
elizabeth 1
embellish 1
encompass 1
english 1
epic 1
epics 2
especially 1
europe 1
european 1
even 1
example 1
experience 2
fashion 1
feelings 1
fiction 4
fictional 2
fictionality 1
flash 1
followed 1
for 1
found 1
france 1
from 1
gallantry 1
gate 1
genre 1
geoffrey 1
golden 1
had 1
hand 2
has 1
historians 1
historical 1
historiography 1
history 1
however 3
important 1
in 12
include 2
including 1
individualistic 1
intimate 4
invent 1
inventions 1
is 4
its 1
japan 1
juan 1
language 1
late 1
leigh 1
length 6
life 1
literary 2
longest 1
lord 1
middle 1
modern 3
more 2
most 2
narrative 2
narratives 2
new 1
not 2
notion 1
novel 7
novella 1
novels 4
odyssey 1
of 18
often 1
on 2
onegin 2
opinion 1
or 1
order 1
other 2
passage 1
performance 1
period 2
personal 2
place 1
plays 1
political 1
possible 1
precise 1
problematic 1
prose 6
purposes 1
pushkin 1
ranking 1
rather 1
reading 1
realities 1
recent 1
recited 1
requirement 1
rival 1
romance 3
rooted 1
s 5
saw 1
secret 1
select 1
seth 1
short 2
should 1
situations 1
social 1
southern 1
speeches 1
spread 1
standard 1
stanzas 1
still 1
story 1
such 1
system 1
tales 1
text 1
th 5
than 2
that 1
the 28
theaters 1
these 1
this 2
those 1
though 1
throughout 1
to 3
today 1
totality 1
traditional 1
traditionally 1
troyes 1
types 1
use 1
verse 5
views 1
vikram 1
was 1
while 1
with 4
works 1
world 1
would 2
yevgeniy 1
